### DISCUSSION TITLE
multiple settings menu

### DISCUSSION THREAD

**Discussion by Thijsvanzwam** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2612] [Ideas]
I use two different print beds. One is PEI and is used for printing PLA, PETG etc. the other is non-PEI and is used for printing TPU.
The bedsensor to set the z-axis to zero reacts different on the different print beds.
I have to manually change the nozzle offset each time I change the bed sheet.
Is there a way of setting up two user modes in which I can store two different offset settings?

**Comment by kisslorand** [comment]
You can define two separate custom gcodes in the "config.ini" for your two settings and just use those.

**Comment by Thijsvanzwam** [comment]
ok, only... how do I select the code I want to use?
for example:
i load a tpu roll so I want to change some settings in the printer (speed and nozzle offset) can I choose that somewhere on the touch menu and then when loading another filament type, choose the other settings?

**Reply by Thijsvanzwam** [reply]
ah... found it. thanks!
